guides/source/association_basics.md

```diff
@@ -2161,7 +2161,7 @@ The `collection.size` method returns the number of objects in the collection.
 
 ##### `collection.find(...)`
 
-The `collection.find` method finds objects within the collection. It uses the same syntax and options as
+The `collection.find` method finds objects within the table (not an unsaved collection). It uses the same syntax and options as
```

Comment:
```suggestion
The `collection.find` method finds objects within the collection by querying the database. It uses the same syntax and options as
```

> not an unsaved collection

The collection being saved or unsaved  is not relevant, so we can drop that out.

Reply:
I changed it to your wording but added explicitly "It will not find  objects that are not in the table" because "querying the table" does not necessarily imply _not_ finding items in the collection but not in the table, and being explicit is useful anyway.